fake_joint_driver/src/fake_joint_driver_node.cpp

```diff
@@ -31,7 +31,7 @@ int main(int argc, char **argv)
 
   while (ros::ok())
   {
-    robot.update();
+    robot.update(ros::Duration(0.010));
```

Comment:
We've started a convention where hard-coded values go in constants at the top of C++ files. Makes them easier to find. So, at line 12, put this:

namespace {  // ensure this constant stays local to this file
constexpr double CONTROL_PERIOD = 0.01;
}

At line 28:  `ros::Rate rate(1.0 / ros::Duration(CONTROL_PERIOD).toSec());`
At line 34: `robot.update(ros::Duration(CONTROL_PERIOD));`

Reply:
https://github.com/PickNikRobotics/fake_joint/pull/1/commits/b929033452c6de2d19c22bccb9addb16fafc8d3e